docker-compose.yml

```diff
@@ -178,6 +178,7 @@ services:
     image: composer:latest
     volumes:
       - ./service-front:/app
+      - ./shared/logging:/shared/logging
```

Comment:
We could do, however, that would mean that if when running locally, you cd into service-pdf for example some composer commands would no longer work because, at this point, the logging code is not contained within this directory.  By leaving it relative, the directory structure is the same when running local as when running in CI, which would seem good (always good for local and CI to resemble each other as much as poss).  That was my reasoning anyway. :)